core/cli/src/plugin/resolve-root.ts

```diff
@@ -1,10 +1,26 @@
 import path from "path"
+import { resolve } from "./require-resolve"
+import { ToolKitError } from "@dotcom-tool-kit/error"
+import { styles } from "@dotcom-tool-kit/logger"
 
 export function resolveRoot(id: string, root: string): string {
+  const isRelative = id.startsWith('./')
```

Comment:
previously we were checking for absolute paths here as well, but they're preserved by `path.join`, it's only `./` relative paths that aren't